modules/homefinder/widgets/class-homes-for-sale-widget.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * HomeFinder Homes for Sale Widget (http://www.homefinder.com/widgets/)
+ *
+ * @package RE-PRO
+ */
+
+	/* Exit if accessed directly. */
+if ( ! defined( 'ABSPATH' ) ) { exit; }
+
+/**
+ * HomeFinderHomesForSale class.
+ *
+ * @extends WP_Widget
+ */
+class HomeFinderHomesForSale extends WP_Widget {
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)
* Opening brace of a class must be on the line after the definition

<!-- Comment by [SideCI](https://sideci.com) -->